cql3/result_generator.hh

```diff
@@ -114,7 +114,7 @@ private:
             }
         }
 
-        uint32_t rows_read() const { return _total_row_count; }
+        auto rows_read() const { return _total_row_count; }
```

Comment:
This is a bad idea, as it moves the interface of the function into its implementation. The implementation is trivial, but still it's moving the code in a JavaScript-like direction.

We should be able to terminate questions about types in the function declaration.

Reply:
Ok, fixed.